index/indexfile.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build linux || darwin
+//go:build !windows
```

Comment:
Small comment, this file could be named `indexfile_unix.go` for consistency with the others.